README.md

```diff
@@ -19,33 +19,62 @@ charts/
 
 This repository branch contains a `configuration.yaml` file that is used to specify how it interacts with other repository branches.
 
-#### Sync
-
-This branch syncs with the generated assets from the following branches:
-- dev-v2.5 at https://github.com/rancher/charts.git (only latest assets)
-
-To release a new version of a chart, please open the relevant PRs to one of these branches. 
-
-Merging should trigger a sync workflow on pushing to these branches.
-
 ### Cutting a Release
 
-In the Live branch, cutting a release requires you to run the `make sync` command.
-
-This command will automatically get the latest charts / resources merged into the the branches you sync with (as indicated in this branch's `configuration.yaml`) and will fail if any of those branches try to modify already released assets.
-
-If the `make sync` command fails, you might have to manually make changes to the contents of the Staging Branch to resolve any issues.
+In the Live branch, cutting a release requires you to copy the contents of the Staging branch into your Live Branch, which can be done with the following simple Bash script.
+
+```bash
+# Assuming that your upstream remote (e.g. https://github.com/rancher/charts.git) is named `upstream` 
+# Replace the following environment variables
+STAGING_BRANCH=dev-v2.x
+LIVE_BRANCH=release-v2.x
+FORKED_BRANCH=release-v2.x.y
+git fetch upstream
+git checkout upstream/${LIVE_BRANCH} -b ${FORKED_BRANCH}
+git branch -u origin/${FORKED_BRANCH}
+git checkout upstream/${STAGING_BRANCH} -- charts assets index.yaml
+git add charts assets index.yaml
+git commit -m "Releasing chart"
+git push --set-upstream origin ${FORKED_BRANCH}
```

Comment:
Nit: I would not put origin here as it points out to the rancher/charts repo. I would expect release caption to push to their fork first then create pull requests. I don't think we want forked branch to be pushed accidently on rancher/charts.

Reply:
@StrongMonkey above it says `# Assuming that your upstream remote (e.g. https://github.com/rancher/charts.git) is named upstream`, so I think the expectation is that origin is their fork.

We can modify the documentation for the new description, but I think that needs to be done as a rancher/charts-build-scripts change to the template that we can then pull in.

Reply:
ok that's fine.

Reply:
Just make sure we can call it out in confluence. Mostly likely people have origin pointing upstream git repo.